### ISSUE TITLE
.mm3d read bug

### ISSUE THREAD

**Issue by zturtleman** [zturtleman/mm3d#113]
.mm3d read bug: https://github.com/zturtleman/mm3d/blob/f21f69ae33cbee6f3a93de83ac8f0765e059934c/src/libmm3d/mm3dfilter.cc#L1572

_Originally posted by @mick-p1982 in https://github.com/zturtleman/mm3d/issues/108#issuecomment-571800540_

**Comment by mick-p1982** [comment]
Another one...

https://github.com/zturtleman/mm3d/blob/ebbf3bb1951aa5f7e3617a2bce208fb5bf98b91d/src/implui/viewwin_influences.cc#L318

**Comment by mick-p1982** [comment]
mergeModels
```
	if(frameAnimsNeeded)
	{
		setFrameAnimVertexCount(m_vertices.size());
		setFrameAnimPointCount(m_vertices.size());
	}
```

**Comment by zturtleman** [comment]
Thanks.

1. Fixed in ed2ba65de1f9622142bd451aa93470d241967db9 and 8e16bb8da768f4a8be546f848ac0291ffb57e209
2. Fixed in fa24cabf29e23aae8e5cfa475c81f8e33f299f60
3. Fixed in c91325fade91a5669ad1cdca742a0767d610b0f7
